server/game/core/Game.js

```diff
@@ -26,7 +26,8 @@ const { cards } = require('../cards/Index.js');
 // const ConflictFlow = require('./gamesteps/conflict/conflictflow');
 // const MenuCommands = require('./MenuCommands');
 
-const { EventName, ZoneName, TokenName, Trait, WildcardZoneName } = require('./Constants.js');
+const { EventName, ZoneName, Trait, WildcardZoneName, TokenUpgradeName, TokenUnitName } = require('./Constants.js');
+const { BaseStepWithPipeline } = require('./gameSteps/BaseStepWithPipeline.js');
```

Comment:
Is this import needed? Normally I wouldn't care but if the really basic classes import each other that creates more possibility for import loops